docs/i18n.stories.mdx

```diff
@@ -0,0 +1,52 @@
+<Meta title="Documentation/i18n" />
+
+# i18n
+
+## Default
+
+By default `carbon` uses `en-GB` translations. **No action required**.
+
+Preview [src/locales/en-gb.js](https://github.com/Sage/carbon/blob/master/src/locales/en-gb.js)
```

Comment:
This link appears to re-direct to an `error 404` page.

Reply:
Yes, you are right but this file will available there when pull request is merged.

Reply:
Learn something new everyday 😄